recipes/recipes_emscripten/udunits2/recipe.yaml

```diff
@@ -0,0 +1,43 @@
+context:
+  version: "2.2.28"
+
+package:
+  name: udunits2
+  version: "{{ version }}"
+
+source:
+  url: https://artifacts.unidata.ucar.edu/repository/downloads-udunits/{{ version }}/udunits-{{ version }}.tar.gz
+  sha256: 590baec83161a3fd62c00efa66f6113cec8a7c461e3f61a5182167e0cc5d579e
+  patches:
+    - 0001-static-path-xml.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - "{{ compiler('c') }}"
+    - make  # [not win]
+    - autoconf  # [not win]
+    - automake  # [not win]
+    - libtool  # [not win]
+    - texinfo  # [not win]
+  host:
+    - expat
+  run:
+    - expat
+
+# test:
+#   commands:
+#     - udunits2 -h
+#     - udunits2 -H meter -W miles
```

Comment:
@wolfv you asked me to write a test in https://github.com/emscripten-forge/recipes/pull/152#issuecomment-1206474268 but these are commented out. What happened here? Should I try to re-enable them?

Reply:
Hey @ocefpaf sorry for the confusion. We actually use a custom test thingy because the tests need to run in a weird environment (inside NodeJS etc.).

E.g. it looks like:

```yaml
extra:
  emscripten_tests:
    python:
      pytest_files:
        - test_cftime.py
```

Where `test_cftime.py` is a file in the same folder as `recipe.yaml`. You can find the cftime example here: https://github.com/emscripten-forge/recipes/blob/main/recipes/recipes_emscripten/cftime/recipe.yaml#L39-L43

If you want to add some more tests, feel free! :)